node/src/env.rs

```diff
@@ -0,0 +1,13 @@
+use tailcall::EnvIO;
+
+pub struct WasmEnv {}
+impl WasmEnv {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+impl EnvIO for WasmEnv {
+    fn get(&self, key: &str) -> Option<String> {
+        unimplemented!("Not needed for npm pkg")
```

Comment:
Not ok. There should be a way to support this in the browser.

Reply:
On node we do have process.env, ideally we should read from it